### ISSUE TITLE
Can't run amok on osx

### ISSUE THREAD

**Issue by EatYourSpinach** [caspervonb/amok#26]
I get the following error, and have set my environment variables as described on the home page:
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: spawn /Applications/Google\ ENOENT
    at exports._errnoException (util.js:746:11)
    at Process.ChildProcess._handle.onexit (child_process.js:1046:32)
    at child_process.js:1137:20
    at process._tickCallback (node.js:355:11)

**Comment by caspervonb** [comment]
This is a bug, due to how spaces are handled, patch will be coming real soon.

**Comment by caspervonb** [comment]
@EatYourSpinach Surround the absolute path with double quotes, should take care of it.

**Comment by EatYourSpinach** [comment]
I'm still getting this error when running: amok dist/bundle.js
First I ran: export AMOK_BROWSER='"/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --remote-debugging-port=9222"'
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: spawn "/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --remote-debugging-port=9222" ENOENT
    at exports._errnoException (util.js:746:11)
    at Process.ChildProcess._handle.onexit (child_process.js:1046:32)
    at child_process.js:1137:20
    at process._tickCallback (node.js:355:11)

**Comment by caspervonb** [comment]
I think this form *should* work.

```sh
export AMOK_BROWSER='"/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome" --remote-debugging-port=9222'
```

**Comment by caspervonb** [comment]
Actually, no it was a bad idea to rush out the commit before calling it a night. 4aec664dbdf09bb0139890bb0b9a5f0749d70593 breaks option parsing

**Comment by caspervonb** [comment]
*Should* be fixed in 0.9.2

**Comment by caspervonb** [comment]
Tracking this in #31 from now on until it is truly resolved.
